cmd/createorupdate/createorupdate.go

```diff
@@ -55,6 +56,9 @@ func createOrUpdate(ctx context.Context, oc *v20180930preview.OpenShiftManagedCl
 		log.Info("merge old and new config")
 		p.MergeConfig(ctx, cs, oldCs)
 	}
+	logs := &logrus.Entry{}
+	hc := healthcheck.NewSimpleHealthChecker(logs)
+	hc.HealthCheck(ctx, oldCs)
```

Comment:
I don't understand why this is here?

Reply:
You are right.  Will remove.

This was for testing.  I had to find out the hard way that my yaml library wasn't the same as the `createorupdate.go`.  I placed this code here to get the proper data so that I could reproduce.